saagieapi/env_vars/env_vars.py

```diff
@@ -257,10 +269,10 @@ def update_for_project(
             When the variable doesn't already exist
         """
 
-        existing_env_var = self.list_for_project(project_id)["projectEnvironmentVariables"]
+        existing_env_var = self.list_for_project(project_id, pprint_result=False)["projectEnvironmentVariables"]
 
         if name not in [env_var["name"] for env_var in existing_env_var]:
-            raise ValueError("Environment variable does not exists")
+            raise ValueError("❌ Environment variable does not exists")
```

Comment:
Maybe it's better to specify the name that doesn't exist as environment variable like:

`f"❌ Environment variable {name} does not exists"`

Reply:
done